nextflow_schema.json

```diff
@@ -281,7 +288,11 @@
         },
         "aligner": {
             "type": "string",
-            "default": "bwamem2"
+            "default": "bwa",
+            "fa_icon": "fas fa-puzzle-piece",
+            "enum": ["bwa", "bwamem2", "dragmap"],
+            "description": "Specify aligner to be used to map reads to reference genome.",
+            "help_text": "TODO"
```

Comment:
:D nice, the help message is a TODO

Reply:
I copied the help from Sarek, but it was Sarek specific